db/seeds.rb

```diff
@@ -40,7 +61,11 @@
 
 Rental.create!([
   {user_id: 2, days_rented: 5, total_price: 310, status: 'Completed'},
-  {user_id: 2, days_rented: 3, total_price: 150, status: 'Completed'}
+  {user_id: 2, days_rented: 3, total_price: 150, status: 'Completed'},
+  {user_id: 3, days_rented: 3, total_price: 150, status: 'Paid'},
+  {user_id: 4, days_rented: 3, total_price: 150, status: 'Cancelled'},
+  {user_id: 2, days_rented: 3, total_price: 150, status: 'Out'},
+  {user_id: 2, days_rented: 3, total_price: 150, status: 'Paid'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.